src/main/java/uk/gov/hmcts/sptribs/caseworker/model/ReferToJudge.java

```diff
@@ -42,4 +45,11 @@ public class ReferToJudge {
         typeOverride = TextArea
     )
     private String additionalInformation;
+
+    @CCD(
+        label = "Referral Date",
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+    private LocalDate referralDate;
```

Comment:
We will not face any application level issues after releasing this enhancement. We can follow one of the following three options to handle the old cases,

1. If we want to let the old cases to not to show referral date, we don't need to do anything. (OR)
2. User can recreate the referral event again and the referral date will be populated with the newly created event date.(OR)
3. We can write an SQL script/query to create referral date with the respective event date for old cases.

Reply:
Agreed on option 2.